timetagger_cli/core.py

```diff
@@ -154,35 +154,56 @@ def stop():
         print_records(running_records)
 
 
-def resume():
-    """Start a timer with the same description as previous."""
+def resume(selected=None):
+    """Start a timer with the same description as the selected record."""
 
     now = int(time.time())
     d = datetime.datetime.fromtimestamp(now)
 
     today = datetime.datetime(d.year, d.month, d.day)
-    tomorrow = today + datetime.timedelta(1)
+    tomorrow = today + datetime.timedelta(days=1)
+    last_week = today - datetime.timedelta(days=7)
 
-    t1 = int(today.timestamp())
+    t1 = int(last_week.timestamp())
     t2 = int(tomorrow.timestamp())
 
-    # Get records from today
+    # Get records from the last week
     records = request("GET", f"records?timerange={t1}-{t2}")["records"]
 
-    if len(records) == 0:
-        print("No records earlier today.")
-        return
-
     # Remove HIDDEN records
     filtered_records = [r for r in records if "HIDDEN" not in r["ds"]]
 
-    # Get last record
-    last_record = filtered_records[-1]
+    if len(filtered_records) == 0:
+        print("No records within the last week.")
+        return
+
+    # Sort records by server time to get same order as on server
+    filtered_records.sort(key=lambda r: r["st"])
```

Comment:
The `st` field is only used for synchronization purposes. Records in the app are shown simply based on their `t1` and `t2`.
```suggestion
    # Sort records by time
    filtered_records.sort(key=lambda r: r["t2"])
```